### PR TITLE
Changes for release v1_3.

### PR DESCRIPTION
**Pull request by Raibaz**

### CODE REVIEW

File: examples/BasicOperations/GetCampaigns.php [hunk 1]
```diff
@@ -98,7 +98,7 @@ public static function runExample(GoogleAdsClient $googleAdsClient, $customerId)
         // Issues a search request by specifying page size.
         $response =
             $googleAdsServiceClient->search($customerId, $query, ['pageSize' => self::PAGE_SIZE]);
-
+
```
Comment (PierrickVoulet) [inline examples/BasicOperations/GetCampaigns.php]: Nit: extra spaces

Comment (Raibaz) [reply examples/BasicOperations/GetCampaigns.php]: Fixed, thanks.

Comment (fiboknacky) [reply examples/BasicOperations/GetCampaigns.php]: It's not fixed yet. There should be no spaces in this line at all. Should be just a blank line, like it's never modified.

**Comment by Raibaz** [comment]
@fiboknacky the GetCampaigns example is now reverted to how it was before the release, you can see it since it doesn't appear in https://github.com/googleads/google-ads-php/pull/88/files

**Pull request merged**